Yoast/javascript
# Add sentence beginnings assessment for Turkish

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds the consecutive sentences beginnings assessment for Turkish.
* [Yoast SEO Free] Adds a consecutive sentences beginnings assessment for Turkish.

## Test instructions

This PR can be tested by following these steps:

Test the following in WP SEO:
* Set the site language to Turkish.
* Write three or more Turkish sentences...
    * that all begin with the same written-out number, like `üç` ("three")
       * E.g.   `Üç sokak kapandı. Üç komşu etkilendi. Üç makale yazıldı.`
    * that all begin with the same demonstrative pronoun, like `bu` ('this').
       * E.g. `Bu tartışma önemlidir. Bu insanlar etkilendi. Bu konular zaman alır.`
* In each of the above cases, the sentence beginning readability assessment should read: `Consecutive sentences: There is enough variety in your sentences. That's great!`
* Write three or more Turkish sentences that begin with the same word.
  * E.g. `Uzmanlar yanılmıştı. Uzmanlar sorumlu tutulacak. Uzmanlar bir özür yayınladı. Uzmanlar daha fazla yorum yapacak.`
* The sentence beginning readability assessment should read: `Consecutive sentences: The text contains 4 consecutive sentences starting with the same word. Try to mix things up!`

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-537
